v21.1/connect-to-the-database.md

```diff
@@ -20,7 +20,7 @@ For a reference that lists all of the supported cluster connection parameters, s
 Do the following:
 
 - Set up a local CockroachDB cluster:
--- [Create](install-cockroachdb.html) and [configure](secure-a-cluster.html) a local cluster.
+  - [Create](install-cockroachdb.html) and [configure](secure-a-cluster.html) a local cluster.
 - [Install a Postgres client](install-client-drivers.html).
```

Comment:
Could we change this to `Install a client driver or ORM framework`? I think that's a little more accurate.
(This applies to the CockroachCloud connect page as well.)